BootloaderCorePkg/Stage1A/Stage1A.c

```diff
@@ -240,6 +240,9 @@ SecStartup2 (
     DEBUG ((DEBUG_INIT, "\n%a\n", mBootloaderName));
   }
 
+  if (Stage1aAsmHob->BistVal != 0)
+    CpuHalt ("CPU BIST failure!\n");
```

Comment:
Minor comments on coding styles.  :)
According to EDK2 coding style, it is required to add curly brace {} after if condition even if it has only single statement after if.  So it needs to be:
if (Stage1aAsmHob->BistVal != 0) {
    CpuHalt ("CPU BIST failure!\n");
}
Aiden also pointed it out in his previous comments.

Reply:
Done